aws-secretsmanager/src/main/java/io/micronaut/aws/secretsmanager/SecretsManagerConfigurationClient.java

```diff
@@ -33,22 +33,38 @@
  */
 @Requires(beans = {
         AwsDistributedConfiguration.class,
-        SecretsManagerKeyValueFetcher.class
+        SecretsManagerGroupNameAwareKeyValueFetcher.class
 })
 @Singleton
 @BootstrapContextCompatible
 public class SecretsManagerConfigurationClient extends AwsDistributedConfigurationClient {
 
+    private SecretsManagerConfiguration secretsManagerConfiguration;
+
     /**
-     *
      * @param awsDistributedConfiguration AWS Distributed Configuration
      * @param secretsManagerKeyValueFetcher Secrets Manager Key Value Fetcher
      * @param applicationConfiguration Application Configuration
+     * @param secretsManagerConfiguration Secrets Configuration
      */
     public SecretsManagerConfigurationClient(AwsDistributedConfiguration awsDistributedConfiguration,
-                                             SecretsManagerKeyValueFetcher secretsManagerKeyValueFetcher,
-                                             @Nullable ApplicationConfiguration applicationConfiguration) {
+                                             SecretsManagerGroupNameAwareKeyValueFetcher secretsManagerKeyValueFetcher,
+                                             @Nullable ApplicationConfiguration applicationConfiguration,
+                                             SecretsManagerConfiguration secretsManagerConfiguration) {
         super(awsDistributedConfiguration, secretsManagerKeyValueFetcher, applicationConfiguration);
```

Comment:
this is a breaking change, retain the original constructor and deprecate it supplying a default value to the new constructor